### PR TITLE
boards/arduino-mkrfox1200: add initial implementation (without SigFox)

### PR DESCRIPTION
**Pull request by aabadie**
This PR adds support for the mkrfox1200 board. It will be possible to use SigFox with it when #7087 is merged.

~~Since #6881 factorize the ports with mkr1000, this PR is based on it.~~

### CODE REVIEW

**Comment by aabadie** [comment]
rebase and conflicts fixed

**Comment by aabadie** [comment]
This PR is ready for review, with #7087 (done in #7093), it can be used to send messages using Sigfox.

File: boards/arduino-mkrfox1200/doc.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
++ *    Example with `hello-world` application:
++ * ```
++ *      make BOARD=arduino-mkrfox1200 -C examples/hello-world flash
++ * ```
++ *
++ * ### Accessing STDIO via UART
++ *
++ * To access the STDIO of RIOT, a FTDI to USB converted needs to be plugged to
++ * the RX/TX pins on the board.
++ */
```
Comment (kYc0o) [inline boards/arduino-mkrfox1200/doc.txt]: I think you have a copy-paste issue here, all lines start by `+`.

Comment (aabadie) [reply boards/arduino-mkrfox1200/doc.txt]: Fixed, directly squashed and rebased

**Comment by kYc0o** [comment]
ACK and go!

**Pull request merged**